_pytest/config.py

```diff
@@ -399,13 +399,15 @@ def consider_conftest(self, conftestmodule):
             self.consider_module(conftestmodule)
 
     def consider_env(self):
-        self._import_plugin_specs(os.environ.get("PYTEST_PLUGINS"))
+        specs = os.environ.get("PYTEST_PLUGINS")
+        if specs:
+            plugins = specs.split(',')
```

Comment:
that may be necessary, and a good thing to do, we have so many inconsistencies ^^